app/config.js

```diff
@@ -2,8 +2,14 @@ import fs from "fs";
 import Store from "electron-store";
 import ini from "ini";
 import { stakePoolInfo } from "./middleware/stakepoolapi";
+import { app } from "electron";
 import { appDataDirectory, getGlobalCfgPath, dcrdCfg, getWalletPath, dcrwalletCfg, getDcrdRpcCert } from "./main_dev/paths";
 
+// setPath as decrediton
+// app is not set after building so we need that check
+if(app)
```

Comment:
Very small nit.

Either add braces ({}) on the whole if or leave it at the same line. And add a single space after the `if`.

Reply:
Also, I'm not sure I'm following the comment ("app is not set after building").

I added a small `console.log` on both branches of the if, and tested after a `yarn dev`, `yarn build/start`, `yarn package-dev-linux` and all three go through the `true` branch (ie, app **is** set).

Can you explain a bit better what you're seeing when app is not set?